Adyen/UI/Form/Items/Basic Items/FormVerticalStackItemView.swift

```diff
@@ -45,3 +50,15 @@ internal final class FormVerticalStackItemView<FormItemType: FormItem>: FormItem
     }
 
 }
+
+extension FormVerticalStackItemView: SelfRenderingFormItemDelegate {
+
+    internal func didUpdatedItems(_ items: [FormItem]) {
+        stackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+        views = items.map(FormVerticalStackItemView.build)
+        views.forEach(stackView.addArrangedSubview)
+        stackView.setNeedsLayout()
+        views.first { $0.canBecomeFirstResponder }?.becomeFirstResponder()
```

Comment:
you need to call `self.updatePreferredContentSize()`in case the height changed , right?

Reply:
hm.. I believe because there are keyboard event - it will be called eventually.
I don't like it, but it works 🤔

Reply:
its fine for now